src/app/shared/log-in/log-in.component.scss

```diff
@@ -0,0 +1,32 @@
+@import '../../../styles/variables.scss';
+
+.form-login {
+  width: 100%;
+  max-width: 330px;
+  padding: 15px;
+  margin: 0 auto;
+}
+.form-login .checkbox {
+  font-weight: 400;
+}
+.form-login .form-control {
+  position: relative;
+  box-sizing: border-box;
+  height: auto;
+  padding: 10px;
+  font-size: 16px;
+}
+.form-login .form-control:focus {
+  z-index: 2;
+}
+.form-login input[type="email"] {
+  margin-bottom: -1px;
+  border-bottom-right-radius: 0;
+  border-bottom-left-radius: 0;
+}
+.form-login input[type="password"] {
+  margin-bottom: 10px;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+}
+
```

Comment:
I'd try avoid writing all this css. You can accomplish most of what `log-in.component.scss` does with a few choice boostrap classes and the end-result will more robust on different screen sizes. Since bootstrap 4 has classes for things like [spacing](https://getbootstrap.com/docs/4.0/utilities/spacing/) now, there are even fewer reasons to write your own css than before.

 e.g. if you change `login-page.component.html` to the following, you're 90% there:
```html
<div class="container">
  <div class="text-center mt-5 row justify-content-md-center">
    <div class="col-sm-6 col-md-4">
      <img class="mb-4" src="assets/images/dspace-logo.png" alt="" width="72" height="72"> 
      <h1 class="h3 mb-0 font-weight-normal">{{"login.form.header" | translate}}</h1>
      <ds-log-in></ds-log-in>
    </div>
  </div>
</div>
```
All this does is add basic grid classes and a few extra divs to put them in.

If you do end up having to write css, try to avoid hardcoded pixel values when possible. Instead derive what you need from existing bootstrap variables. That makes the end result easier to theme, and pre-existing bootstrap themes will automatically work with your components as well. 

The one exception perhaps in the example above are the dimensions of the dspace logo. I'd move those to css, and I wouldn't mind using pixel values for it, though I'd perhaps put them in variables of our own, so that if we need a similar logo later, we can reuse them.